classes/sdcard_image-rpi.bbclass

```diff
@@ -0,0 +1,214 @@
+# Imported from meta-raspberrypi
+# kernel dependency changed to do_webos_deploy_fixup task
```

Comment:
This isn't true right? the only difference is:
https://github.com/agherzan/meta-raspberrypi/pull/1289
and do_webos_deploy_fixup task was removed by "FU" commit now squashed to the next commit.

Reply:
Cleaned that up now, indeed I removed that one to not rely on all the stuff OSE is doing there